test/configs/alias.json

```diff
@@ -0,0 +1,8 @@
+{
+  "file": "alias",
```

Comment:
And frankly speaking, this approach to testing is not very convenient, we need to revise it.